### ISSUE TITLE
Need more Pumpkins in Tirisfal Glades

### ISSUE THREAD

**Issue by JamesGent** [azerothcore/azerothcore-wotlk#4701]
Need more Pumpkins in Tirisfal Glades

##### CURRENT BEHAVIOUR:

8 Pumpkins currently meaning 1 player has to wait ~10 mins for the respawn to do the quest.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Should be about 20+ pumpkins so that 2 players can do the quest together and nobody waits. ClassicWoW has the spawns at 21, and I don't see any patches saying they reduced the amount.

https://classic.wowhead.com/object=375/tirisfal-pumpkin

https://classic.wowhead.com/quest=365/fields-of-grief

In this video you can see the recorded pick up 10 pumpkins with ease, and you can see 3+ other players also picking up pumpkins in the same area at a slower rate.

https://www.youtube.com/watch?v=J3stuaLo25I

##### STEPS TO REPRODUCE THE PROBLEM:

1. Go to Tirisfal Glades
2. Pick up Fields of Grief quest (365)
3. Observe lack of pumpkins.

**Comment by eTzmNcbkrng** [comment]
It's already Blizzlike.
https://www.youtube.com/watch?v=8dNMBWh2wxY&t=4m

**Comment by Zoidwaffle** [comment]
Hey, I had this one recently...

https://github.com/azerothcore/azerothcore-wotlk/pull/4529

back then, it was decided that the current version is correct as pre-Cata.

**Comment by eTzmNcbkrng** [comment]
@Zoidwaffle Yeah I was watching that one when you worked on it, wish I had found this video back then.

**Comment by Si1ker** [comment]
So, has #4529 already resolved this issue?

**Comment by eTzmNcbkrng** [comment]
I don’t think so because all that did was reduce the respond to 5 minutes.
Respawn should be 2 minutes and there should be a lot more than 10 nodes.

**Comment by nicoaravena** [comment]
Actually they are 10 pumpkins, 8 in the field, 2 in the tower (as wotlk was), the respawn time was fixed in #4529.
![imagen](https://user-images.githubusercontent.com/4155059/110050296-65853e80-7d32-11eb-9927-3b9032c1fac8.png)


Also the proposal is from classic blizz server, which was changed from original classic to match with the respawn times and the nodes cuantity from cataclysm.

**Comment by Zoidwaffle** [comment]
Well... Personally, I don't mind aligning the quest more with "it makes sense even if not totally as it was back in the days".

Something like 30 places where they can spawn... and then pool them to have 20 spawns at once with a 5 minute respawn? Or 10 spawns with a 2 min respawn? It would make the game more enjoyable when not playing solo and not break anything IMHO...

TL;DR: I wouldn't mind aligning it more with Classic than Vanilla.

**Comment by Zoidwaffle** [comment]
Dammit, you guys made me make a PR... I didn't have time to test it properly - anyway, it's just a suggestion, it can be closed or accepted -  and I will be happy either way. Let me know if it should be tweaked/improved further.

**Comment by eTzmNcbkrng** [comment]
Hold up, check out the video I linked, it's from Wrath.  
I've been over it a few times, at first I thought the respawn might have been right on 120 seconds, it also look like the game forces there to be 10 nodes active, the 9th one he loots respawns almost immediately.
https://www.youtube.com/watch?v=8dNMBWh2wxY&t=4m - skip to 4 minutes if the timestamp doesn't work, it's not working for me.

**Comment by Zoidwaffle** [comment]
It's hard to say how valid the data is, being potentially poisoned by private servers etc. and so on. But https://www.wowhead.com/object=375/tirisfal-pumpkin#contains indicates more than 10 potential spawning locations - it's at least 20+, but hard to count. Some comments here https://classic.wowhead.com/quest=365/fields-of-grief#comments indicates 10 spawns. So perhaps 20 spawns at once is too much. It's just so much nicer in the updated version...

But I don't know... I really do prefer to keep things Blizz-like, while I also like a bit of artistic freedom when it's not game mechanics or things that break the game (and in this case it's mostly a cosmetic/convenience thing where the more modern version of the quest is just way nicer for an active server with more players).

Anyway, my suggested change can be accepted or discarded - I think it's an upgrade, but it's completely true to say there's evidence towards the quest having different amounts of active pumpkins, number of locations to spawn and different respawn rates at various points of time in the lifetime of WoW.

**Comment by Zoidwaffle** [comment]
It's probably better to take any discussions regarding this here: https://github.com/azerothcore/azerothcore-wotlk/pull/4722.
